data/data-samples/sample-data-oracle-book/src/test/java/io/americanexpress/oracle/test/OracleDataBookIT.java

```diff
@@ -11,10 +11,10 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package io.americanexpress.data.oracle.book.test;
+package io.americanexpress.oracle.test;
```

Comment:
please fix package name
```suggestion
package io.americanexpress.data.oracle.book;
```